src/sentry/search/eap/spans.py

```diff
@@ -344,6 +344,19 @@ def _resolve_search_value(
                         )
                 elif isinstance(value, float):
                     return AttributeValue(val_float=value)
+            elif column_type == constants.BOOLEAN:
+                if operator in constants.IN_OPERATORS:
+                    raise InvalidSearchQuery(
+                        f"{column.public_alias} cannot be used with an IN filter"
+                    )
+                elif isinstance(value, str):
+                    lowered_value = value.lower()
+                    if lowered_value not in constants.BOOLEAN_VALUES:
+                        raise InvalidSearchQuery(
+                            f"{value} is not a valid boolean value, expecting true or false"
+                        )
+                    bool_value = lowered_value in constants.TRUTHY_VALUES
+                    return AttributeValue(val_bool=bool_value)
```

Comment:
Is this going to pass the string `"false"` through to the RPC? Can you make sure that's working as expected as I know snuba just does `bool(value)` to convert the value.

Reply:
Ahh I misread this